### PR TITLE
Rename container to conductor

### PR DESCRIPTION
**Pull request by zippy**
- [x] I have added a summary of my changes to the changelog

Rename Container to Conductor
Rename `holochain_container` to just `holochain`

### CODE REVIEW

**Comment by Connoropolous** [comment]
there would be some follow up work to this, with the npm package rename

File: doc/architecture/decisions/0015-container-api.md [hunk 1]
```diff
@@ -7,39 +7,39 @@ Date: 2018-09-17
 Accepted
 
 ## Glossary
-* **Container**: *We have been struggling to find a term for this concept that feels fitting.
-  In the following, we are using the word *Container*, knowing that this is a temporary placeholder term
+* **Conductor**: *We have been struggling to find a term for this concept that feels fitting.
```
Comment (Connoropolous) [inline doc/architecture/decisions/0015-container-api.md]: These lines should actually change :) now that we've landed this

File: doc/architecture/decisions/0015-container-api.md [hunk 2]
```diff
@@ -7,39 +7,39 @@ Date: 2018-09-17
...
 
-* We have several different roles for using a Holochain app (through a *container*):
+* We have several different roles for using a Holochain app (through a *conductor*):
   * local, QML based UI components
   * as a special case of the above: administration UI
   * externally located (living in a browser) web UIs connected through some form of IPC
   * *bridged* apps, i.e. Holochain app as the user of another Holochain app
   * Services hosting multiple Holochain apps and exposing their zome functions to UIs selectively
-* With HoloSqape and the Holochain-nodejs, we already have two container implementations
+* With HoloSqape and the Holochain-nodejs, we already have two conductor implementations
```
Comment (Connoropolous) [inline doc/architecture/decisions/0015-container-api.md]: This line should change to mention the renamed Rust container, and the renamed nodejs container

File: README.md [hunk 3]
```diff
@@ -27,9 +27,9 @@ This `holochain-rust` repository implements a number of distinct yet overlapping
 
 1. A library for the core Holochain functionality for defining and running DNA instances: [*core*](#core)
 1. A library and syntax for use in Rust based development of Zomes within DNAs, called Holochain Development Kit: [*hdk-rust*](#hdk-rust)
-1. A library for managing instances and connecting them to interfaces: [*container-api*](#container-api)
-1. A Rust based container that uses the container_api: [*container*](#rust-container)
-1. A nodejs based container for running tests: [*nodejs-container*](#nodejs-container)
+1. A library for managing instances and connecting them to interfaces: [*conductor-api*](#conductor-api)
+1. A Rust based conductor that uses the conductor_api: [*conductor*](#rust-conductor)
```
Comment (maackle) [inline README.md]: I think "conductor" in plain English reads better if capitalized.

```suggestion
1. A Rust based Conductor that uses the conductor_api: [*conductor*](#rust-conductor)
```

File: README.md [hunk 4]
```diff
@@ -27,9 +27,9 @@ This `holochain-rust` repository implements a number of distinct yet overlapping
...
+1. A nodejs based conductor for running tests: [*nodejs-conductor*](#nodejs-conductor)
```
Comment (maackle) [inline README.md]:
```suggestion
1. A nodejs based Conductor for running tests: [*nodejs-conductor*](#nodejs-conductor)
```

File: README.md [hunk 5]
```diff
@@ -50,20 +50,20 @@ An HDK for [Assemblyscript](https://github.com/Assemblyscript/assemblyscript) is
...
 
-### Container API
-*Core* only implements the logic for the execution of a single application. Because the Holochain app ecosystem relies on DNA composibility, we need to be able to load and instantiate multiple DNAs.  We call an executable that can do this an *container*.  The first such containers we implemented were the GUI driven [holosqape](https://github.com/holochain/holosqape) and the CLI driven [hcshell](https://github.com/holochain/holosqape#hcshell) container which we used for running javascript based tests.
+### Conductor API
+*Core* only implements the logic for the execution of a single application. Because the Holochain app ecosystem relies on DNA composibility, we need to be able to load and instantiate multiple DNAs.  We call an executable that can do this an *conductor*.  The first such conductors we implemented were the GUI driven [holosqape](https://github.com/holochain/holosqape) and the CLI driven [hcshell](https://github.com/holochain/holosqape#hcshell) conductor which we used for running javascript based tests.
```
Comment (maackle) [inline README.md]:
```suggestion
*Core* only implements the logic for the execution of a single application. Because the Holochain app ecosystem relies on DNA composibility, we need to be able to load and instantiate multiple DNAs.  We call an executable that can do this a *Conductor*.  The first such Conductors we implemented were the GUI driven [holosqape](https://github.com/holochain/holosqape) and the CLI driven [hcshell](https://github.com/holochain/holosqape#hcshell) Conductor which we used for running javascript based tests.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- Rename container to conductor, and `holochain_container` executable to `holochain`
```
Comment (Connoropolous) [inline CHANGELOG.md]:
```suggestion
- Rename the term Container and all references to it to Conductor
- The `holochain_container` executable has been renamed to simply `holochain`

```

**Comment by Connoropolous** [comment]
I'm working on the merge conflicts for this branch
